libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -378,8 +378,14 @@ namespace SohImGui {
                 ImGui::Text("Gameplay");
                 ImGui::Separator();
 
-                if (ImGui::Checkbox("Fast Text", &Game::Settings.enhancements.fast_text)) {
-                    CVar_SetS32("gFastText", Game::Settings.enhancements.fast_text);
+                ImGui::Text("Text Speed", Game::Settings.enhancements.text_speed);
+                if (ImGui::SliderInt("##TEXTSPEED", &Game::Settings.enhancements.text_speed, 1, 5)) {
```

Comment:
what happens if we go higher than 5? does it break? part of me would want to be able to set the max value of the slider to 100 or something using the ini but this is definitely a nitpick at best

overall everything looks great! thanks for this!

Reply:
Higher than 5 works but you very much end up with diminishing returns.

Originally had it at 10 and thought the change wasn't working when I set it to 5, 100 would probably be noticeably different though!

I think for a lot of people 5 is already more than enough, so that feels like the sweet spot, imo.